.github/workflows/test.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Node
         uses: actions/setup-node@v4
         with:
-          node-version: ${{ env.node }}
+          node-version: ${{ vars.NODE_VERSION }}
```

Comment:
Why this change? Also, if we are no longer using `env.node`, we could delete it from the `env`.

Reply:
Moved to organisation level variable.